src/components/AvatarWithDisplayName.js

```diff
@@ -0,0 +1,118 @@
+    const isExpenseReport = ReportUtils.isExpenseReport(props.report);
+    const icons = ReportUtils.getIcons(props.report, props.personalDetails, props.policies);
+    const ownerPersonalDetails = OptionsListUtils.getPersonalDetailsForLogins([props.report.ownerEmail], props.personalDetails);
+    const displayNamesWithTooltips = ReportUtils.getDisplayNamesWithTooltips(ownerPersonalDetails, false);
+    return (
+        <View>
+            <View style={[styles.appContentHeaderTitle]}>
+                {Boolean(props.report && title) && (
+                    <View
+                        style={[
+                            styles.flex1,
+                            styles.flexRow,
+                            styles.alignItemsCenter,
+                            styles.justifyContentBetween,
+                        ]}
+                    >
+                        {isExpenseReport ? (
+                            <SubscriptAvatar
+                                mainAvatar={icons[0]}
+                                secondaryAvatar={icons[1]}
+                                mainTooltip={props.report.ownerEmail}
+                                secondaryTooltip={subtitle}
+                                size={props.size}
+                            />
+                        ) : (
+                            <Avatar
+                                size={props.size}
+                                source={icons[0].source}
+                                type={icons[0].type}
+                                name={icons[0].name}
+                                containerStyles={props.size === CONST.AVATAR_SIZE.SMALL ? styles.emptyAvatarSmall : styles.emptyAvatar}
+                            />
+                        )}
+                        <View style={[styles.flex1, styles.flexColumn]}>
+                            <DisplayNames
+                                fullTitle={title}
+                                displayNamesWithTooltips={displayNamesWithTooltips}
+                                tooltipEnabled
+                                numberOfLines={1}
+                                textStyles={[styles.headerText, styles.pre]}
+                                shouldUseFullTitle={isExpenseReport}
+                            />
+                            {subtitle ? (
+                                <Text
+                                    style={[
+                                        styles.sidebarLinkText,
+                                        styles.optionAlternateText,
+                                        styles.textLabelSupporting,
+                                        styles.pre,
+                                    ]}
+                                    numberOfLines={1}
+                                >
+                                    {subtitle}
+                                </Text>
+                            ) : null }
```

Comment:
If you remove it it will still hold space for it and not center the title

Reply:
<img width="273" alt="Screenshot 2023-05-03 at 10 32 52 AM" src="https://user-images.githubusercontent.com/4741899/235997806-912c78ff-5120-4482-91cf-9af1835cc80e.png">

@grgia just tested and it seems to work fine for me!

Reply:
@grgia can you confirm if removing the ternary works?

Reply:
Removing it does not:  (notice how top header is misaligned).
Also, I _think_ it's preferred to use `condition ? () : null ` rather than `condition && ()` for conditional rendering? But might need to double check that

 <img width="1804" alt="image" src="https://user-images.githubusercontent.com/38015950/236174756-c129f21a-45c9-44a9-b01d-62eaa1088418.png">

Reply:
Thanks for confirming. Let's keep the ternary then.